app/dashboard/models.py

```diff
@@ -1009,6 +1075,61 @@ def is_notification_eligible(self, var_to_check=True):
             return False
         return True
 
+    @property
+    def bounty(self):
+        try:
+            return Bounty.objects.current().filter(
+                github_url__iexact=self.github_url,
+                network=self.network).order_by('-web3_created').first()
+        except Bounty.DoesNotExist:
+            return None
+
+    def payout_to(self, address, amount_override=None):
+        # TODO: deprecate this after v3 is shipped.
+        from dashboard.utils import get_web3
+        from dashboard.abi import erc20_abi
+        if not address or address == '0x0':
+            raise Exception('bad forwarding address')
```

Comment:
For these generic `Exception` raises, can we specify a custom exception and/or return blank values opposed to raising the exception in our class methods?

Reply:
sure changing that now